assets/components/radioToggle/radioToggle.jsx

```diff
@@ -65,4 +79,5 @@ export default function RadioToggle(props: PropTypes) {
 RadioToggle.defaultProps = {
   checked: '',
   id: null,
+  accessibilityHint: null,
```

Comment:
Shouldn't this be an empty string? `typeof null === 'object'`

Reply:
Thinking about it, this probably shouldn't be optional anyway.

Reply:
Decided to keep as optional and changed to empty string